src/main/scala/uk/gov/nationalarchives/tdr/api/http/GraphQLServer.scala

```diff
@@ -25,7 +25,11 @@ import scala.util.{Failure, Success}
 object GraphQLServer {
 
   val exceptionHandler = ExceptionHandler {
```

Comment:
Something else we need to do is to log the errors caught here so that we don't throw away the original exception information.

We haven't set up any logging yet, so I might add a card to the backlog rather than doing it as part of my current card.